package com.tss.banking.repositary;

import org.springframework.data.jpa.repository.JpaRepository;

import com.tss.banking.entity.Account;

public interface AccountRepository extends JpaRepository<Account,Long>{

}
